web-ng/src/index.html

```diff
@@ -25,7 +25,6 @@
   <link rel="icon" type="image/x-icon" href="favicon.ico">
   <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500&display=swap" rel="stylesheet">
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
-  <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyCYotwGpr2QvwHgCGCMtGC6sDJZLNN6iv4"></script>
```

Comment:
This value of key was causing error

Reply:
That's very surprising! I overlooked this in #231, great catch @parulraheja98 .